api/python/provisioner/salt.py

```diff
@@ -146,9 +151,11 @@ class SaltRunnerArgs(SaltArgsMixin):
     )
     nowait: bool = False
     kw: Dict = attr.Factory(dict)
+    secure: bool = False
```

Comment:
Should the default value of secure not be True?

Reply:
No, it would impact many places where we don't need that.